### PR TITLE
fix(typos): fix typos show up in docs

### PR DESCRIPTION
**Pull request by pearmini**

### CODE REVIEW

**Review by montoyamoraga — APPROVED** [review]
thanks so much!

**Comment by montoyamoraga** [comment]
@all-contributors please add @pearmini for docs

**Comment by pearmini** [comment]
> thanks so much!

You are welcome :)

File: src/core/main.js [hunk 1]
```diff
@@ -377,7 +377,7 @@ class p5 {
         !this._loop ||
         time_since_last >= target_time_between_frames - epsilon
       ) {
-        //mandatory update values(matrixs and stack)
+        //mandatory update values(matrixes and stack)
```
Comment (limzykenneth) [inline src/core/main.js:380]:
```suggestion
        //mandatory update values(matrices and stack)
```
This should be `matrices` if it is plural or `matrix` if it is singular. Not sure whether it is singular or plural though.

**Review by limzykenneth — CHANGES_REQUESTED** [review]

File: src/core/main.js (hunk 1 above)
Comment (montoyamoraga) [reply src/core/main.js:380]: i looked it up here and both matrixes and matrices are correct?
https://thewordcounter.com/plural-of-matrix/
:)

Comment (limzykenneth) [reply src/core/main.js:380]: Hmm...well in that case if this is referring to a plural case we can accept as is then.

Comment (pearmini) [reply src/core/main.js:380]: So now sshould I change matrixes to matrices?

Comment (limzykenneth) [reply src/core/main.js:380]: No worries, it's fine. I'll merge as is. Thanks!

**Pull request merged**